src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2097,7 +2097,7 @@ private function registerRetryableHttpClient(array $options, string $name, Conta
 
         $container
             ->register($name.'.retryable', RetryableHttpClient::class)
-            ->setDecoratedService($name, null, -10) // lower priority than TraceableHttpClient
+            ->setDecoratedService($name)
```

Comment:
to ensure proper order of decoration, we would need to guarantee that it has a higher priority

Reply:
switched to `10`